sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -324,20 +318,19 @@ or
 
     $ pip install ipywidgets_bokeh
 
-Then you can use an IPyWidget in Bokeh, by simply wrapping it in ``IPyWidget``
-model and adding the wrapper to a document or including it in a layout. Given
-that this is run outside Jupyter, there is no need for installing and/or
-enabling any extensions.
+This extension lets you use `IPyWidgets`_ in Bokeh. Simply wrap a widget in an
+``IPyWidget`` model and add the wrapper to a document or include it in a layout.
+You don't have to install or enable any other extensions.
 
 Example
 ~~~~~~~
 
-Suppose we would like to create an application with a single Jupyter slider
-and log its value to the console, as the slider is manipulated. We start by
-constructing the widget and configuring an observer, the same as we would
-do in Jupyter:
+Let's create an example application with a single Jupyter slider that logs its
```

Comment:
@tcmetzger are we preferring "you" to "we/us" in the docs as a guideline now?

Reply:
> Address the reader in the second person (‘you’). Avoid using the first person (‘we’) or the third person (‘the user’).

According to the Style guide, yes.

Reply:
Yes, please always address the reader with a form of "you", according to https://developers.google.com/style/person.

Reply:
Speaking of which, do you think the following paraphrase would be better here?

> For example, try building an application with a single Jupyter slider that logs its adjustments to the console: ...

Reply:
Yes, I think so. After all, "Let's" or "Let us" would not be second person either... 😀

How about:

> Follow these steps to build an application with a single Jupyter slider that ...

This way we wouldn't need the word "example" again, right after the headline.

Reply:
Yeah, I was thinking how to avoid repeating "example" there. This looks a lot better, thanks 👍